File: Ej2cond.java
package ej2cond;

import javax.swing.JOptionPane;

class Ej2cond {

	public static void main(String[] args) {
		int a, b, c;
		a = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		b = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		c = Integer.parseInt(JOptionPane.showInputDialog(null, "ingrese numero"));
		if (a <= b && a <= c) {
			if (b <= c) {
				JOptionPane.showMessageDialog(null, "el orden es: " + a + "-" + b + "-" + c);
			} else {
				JOptionPane.showMessageDialog(null, "el orden es: " + a + "-" + c + "-" + b);
			}
		} else {
			if (b <= a && b <= c) {
				if (a <= c) {
					JOptionPane.showMessageDialog(null, "el orden es: " + b + "-" + a + "-" + c);
				} else {
					JOptionPane.showMessageDialog(null, "el orden es: " + b + "-" + c + "-" + a);
				}
			} else {
				if (a <= b) {
					JOptionPane.showMessageDialog(null, "el orden es: " + c + "-" + c + "-" + b);
				} else {
					JOptionPane.showMessageDialog(null, "el orden es: " + c + "-" + b + "-" + a);
				}
			}
		}
	}

}

File: Ej2cond.js
let a = parseInt(prompt("Ingrese numero"));
let b = parseInt(prompt("Ingrese numero"));
let c = parseInt(prompt("Ingrese numero"));

if (a <= b && a <= c) {
    if (b <= c) {
        alert("El orden es: " + a + "-" + b + "-" + c);
    } else {
        alert("El orden es: " + a + "-" + c + "-" + b);
    }
} else if (b <= a && b <= c) {
    if (a <= c) {
        alert("El orden es: " + b + "-" + a + "-" + c);
    } else {
        alert("El orden es: " + b + "-" + c + "-" + a);
    }
} else {
    if (a <= b) {
        alert("El orden es: " + c + "-" + a + "-" + b);
    } else {
        alert("El orden es: " + c + "-" + b + "-" + a);
    }
}

